doc/nrf/known_issues.rst

```diff
@@ -341,6 +341,22 @@ Immutable bootloader board restrictions
 Build system
 ============
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
+KRKNWK-7827: Application build system is not aware of the settings partition
+  The application build system is not aware of partitions, including the settings partition, which for big applications can result in application code overlapping with other partitions.
+  As a consequence, writing to overlapping partitions might remove or damage parts of the firmware, which can lead to errors that are difficult to debug.
+
+  **Workaround:** Define and use a code partition.
+  You can use one of the following options:
+
+  * :ref:`partition_manager` from |NCS|.
+    For example, for single image, you can use :option:`CONFIG_PM_SINGLE_IMAGE`.
```

Comment:
Two options need to be used here:
```
CONFIG_PM_SINGLE_IMAGE=y
CONFIG_ PM_PARTITION_SIZE_SETTINGS_STORAGE=<settings_value>
```

Calculating the `settings_value` is a bit tricky.
It should take the smaller value of:
`CONFIG_SETTINGS_NVS_SECTOR_COUNT`
and the value provided in the board dts file, e.g. ``:
```
storage_partition: partition@7a000 {
	label = "storage";
	reg = <0x0007A000 0x00006000>;
};
```

So the min between value of CONFIG_SETTINGS_NVS_SECTOR_COUNT multiplied by 0x1000 and, in this case, 0x6000 should be taken.